### DISCUSSION TITLE
Better language switcher

### DISCUSSION THREAD

**Discussion by JohnUW42** [nunocoracao/blowfish#540] [Ideas]
Hi!

It would be nice to have a better language switcher with a drop-down menu.

The attached image is the present implementation — please consider how would it look if there were, for example, 5 or more languages.

![Screenshot](https://user-images.githubusercontent.com/87112040/220812216-4f7c471d-78b4-4944-ba99-11b78cebb339.png)

Below are some examples of Hugo implementations of a drop down-menu language switchers:

1) https://github.com/apvarun/blist-hugo-theme

demo @ https://blist.vercel.app/

2) https://github.com/moodlebox/moodlebox.net

site @ https://moodlebox.net/

3) https://dev.to/feministclickback/how-to-add-a-language-switcher-to-your-hugo-site-2oh6

Thank you!

**Comment by nunocoracao** [comment]
@JohnUW42 created an issue for this one... thanks for the suggestion

**Reply by nunocoracao** [reply]
@JohnUW42 check #542 for some screenshots

**Reply by nunocoracao** [reply]
solved in https://github.com/nunocoracao/blowfish/pull/543

**Comment by JohnUW42** [comment]
Thank you @nunocoracao !

Attached is a screenshot of the present look of the language switcher.

Please note the hover-over message, when hovered over EN, which says "_Welcome to Blowfish! :tada:_".

For the sake of accessibility, do you think that the message could be changed to:

- "Switch to English language" for EN
- "Mude para o idioma português" for PT
- and so on

This would then mean that this must to be defined in a config.

Thank you!



![Screenshot 2023-02-26](https://user-images.githubusercontent.com/87112040/221390414-ca431031-6394-4197-af1b-830e46c6f3d9.png)

**Reply by nunocoracao** [reply]
That is already possible, check Hugo docs for I18n

**Reply by nunocoracao** [reply]
If there is something new which is not supported happy to review a PR for it.
